homeassistant/components/switchbee/entity.py

```diff
@@ -56,3 +58,28 @@ def __init__(
                 f"{coordinator.api.name} ({coordinator.api.mac})",
             ),
         )
+
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self._is_online and super().available
+
+    async def async_refresh_state(self) -> None:
```

Comment:
I think that you can also add `_handle_coordinator_update`, and a stub for `_update_from_coordinator`
Please also remove/adjust the corresponding code from `switch`.

```suggestion
    @callback
    def _handle_coordinator_update(self) -> None:
        """Handle updated data from the coordinator."""
        self._update_from_coordinator()
        super()._handle_coordinator_update()

    @callback
    def _update_from_coordinator(self) -> None:
        """Update the entity attributes from the coordinator data."""
        
    async def async_refresh_state(self) -> None:
```

Reply:
Cancel that, don't add `_handle_coordinator_update`, and `_update_from_coordinator`.
Please do remove/adjust the corresponding code from `switch`.